src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRent = 40;
```

Comment:
it's better to use UPPER_CASE for static variables

Reply:
not fixed